flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/table.scala

```diff
@@ -158,36 +158,60 @@ class Table(
 
   /**
     * Creates [[TemporalTableFunction]] backed up by this table as a history table.
-    *
-    * @param timeIndicator field for the [[TemporalTableFunction]]. Must points to a time indicator
-    * @param primaryKey    field for the [[TemporalTableFunction]].
-    * @return [[TemporalTableFunction]]
+    * Temporal Tables represent a concept of a table that changes over time and for which
+    * Flink keeps track of those changes. [[TemporalTableFunction]] provides a way how to access
```

Comment:
"join" instead of "access"? Because we can really access the data in other operations.

Reply:
I would keep `access` because we have in the backlog story that would enable to write following query:
```
SELECT * FROM Rates('2017/06/14 14:23:12`);
```